keel-api/src/main/kotlin/com/netflix/spinnaker/keel/rest/ArtifactController.kt

```diff
@@ -62,7 +61,7 @@ class ArtifactController(
   fun listVersions(
     @PathVariable name: String,
     @PathVariable type: ArtifactType
-  ): List<DeliveryArtifactVersion> =
+  ): List<String> =
```

Comment:
Simplified this signature so we’re just storing & returning `appversion` strings.